docs/api_openapi_specification.yml

```
@@ -10,7 +10,7 @@ info:
     Worth noting that onboarding aspects of conversation (Informing user of
     limitations and accepting) are not included as within GOV.UK Chat these
     are a UI construct and not a part of core domain constructs.
-  version: "0.1.0"
+  version: "0.1.1"
```

```suggestion
  version: "0.2.0"
```

Probably a minor version bump as we've added a feature.